pkg/model/policymanager/model.go

```diff
@@ -4,22 +4,13 @@
 package policymanager
 
 import (
+	api "fybrik.io/fybrik/manager/apis/app/v1alpha1"
 	"fybrik.io/fybrik/pkg/model/datacatalog"
 	"fybrik.io/fybrik/pkg/model/taxonomy"
 )
 
-// +kubebuilder:validation:Enum=read;write;delete
-type RequestActionType string
-
-// List of operationType
-const (
-	READ   RequestActionType = "read"
-	WRITE  RequestActionType = "write"
-	DELETE RequestActionType = "delete"
-)
-
 type RequestAction struct {
-	ActionType         RequestActionType           `json:"actionType"`
+	ActionType         api.DataFlow                `json:"actionType"`
```

Comment:
`DataFlow` definitions should be moved to `pkg/model/taxonomy/module.go` and defined as 
```
// +kubebuilder:validation:Enum=read;write;delete;copy
type DataFlow string

const (
	// ReadFlow indicates a data set is being read
	ReadFlow DataFlow = "read"

	// WriteFlow indicates a data set is being written
	WriteFlow DataFlow = "write"

	// DeleteFlow indicates a data set is being deleted
	DeleteFlow DataFlow = "delete"

	// CopyFlow indicates a data set is being copied
	CopyFlow DataFlow = "copy"
)
```

Reply:
Done.  Thanks!